assets/js/atomic/blocks/product-elements/sku/index.ts

```diff
@@ -17,11 +17,16 @@ import {
 } from './constants';
 
 const blockConfig: BlockConfiguration = {
-	...sharedConfig,
```

Comment:
Just want to confirm we don't need this here, as we're spreading/merging the same config object in the `registerExperimentalBlockType` function below.

It seems redundant to spread it here in this definition, unless I'm missing something.